MVVM/ViewModel/RoverMode_UIOverlay.cs

```diff
@@ -19,6 +21,36 @@ public partial class RoverMode_UIOverlay : UIOverlay
 	public Task ControlModeChangedSubscriber(MqttClasses.ControlMode newMode)
 	{
 		ControlMode = (int)newMode;
+		CallDeferred(MethodName.UpdateSafeModeIndicatator);
 		return Task.CompletedTask;
 	}
+
+	public override void _Ready()
+	{
+		base._Ready();
+		LocalSettings.Singleton.Connect(LocalSettings.SignalName.PropagatedPropertyChanged,
+			Callable.From<StringName, StringName, Variant, Variant>(OnSettingsPropertyChanged));
+	}
+
+	void OnSettingsPropertyChanged(StringName category, StringName name, Variant oldValue, Variant newValue)
+	{
+		if (category != nameof(LocalSettings.SpeedLimiter))
+			return;
+
+		UpdateSafeModeIndicatator();
+	}
+
+	void UpdateSafeModeIndicatator()
+	{
+		if (ControlMode == 1 && LocalSettings.Singleton.SpeedLimiter.Enabled == true)
+		{
+			SafeModeIndicator.Visible = true;
+			SafeModeIndicator.Text = $"Safe Mode ON - {(LocalSettings.Singleton.SpeedLimiter.MaxSpeed).ToString("P0")}";
```

Comment:
This is the way:
```
SafeModeIndicator.Text = $"Safe Mode ON - {LocalSettings.Singleton.SpeedLimiter.MaxSpeed:P0}";
```

Reply:
Also wth? 
![image](https://github.com/user-attachments/assets/888f57bf-b568-47d5-9b2b-38a581aa5643)
this is a bool already so u do not have to compare it with another value (bool in this case)

Reply:
Both adressed in [1c7a46c](https://github.com/wisniax/RoverControlApp/pull/16/commits/1c7a46caf7707993795a390e68fa60fa18eff9c3)